tools/jit/gen_jit_dispatch.py

```diff
@@ -404,6 +404,7 @@ def declkey(decl):
         ('strides', 'aten::strides(Tensor self) -> int'),
         ('dim', 'aten::dim(Tensor self) -> int'),
         ('numel', 'aten::numel(Tensor self) -> int'),
+        ('element_size', 'aten::element_size(Tensor self) -> int'),
```

Comment:
@bhosmer, is there where we should be putting all of the fast-path functions ? https://github.com/pytorch/pytorch/issues/30774

Reply:
Yep, I don't know the history but this is the current collecting spot AFAIK

Reply:
Re #30774 yeah, this adds element_size to the list - I don't know that it's a tipping point though - and it might be a bit before the unification happens. What's your take though?

Reply:
Yea i think it's fine to add it. It might be nice in some future time to make this list more discoverable (I didn't know about it) via native declaration, but I don't think that should block increasing parity in the meantime.